RELEASES.md

```diff
@@ -7,6 +7,14 @@ Releases are proposed and adopted or rejected using the usual [project governanc
 An anti-pattern that we want to avoid is heavy development or discussions "late cycle" around major releases.
 We want to build a community that is involved and communicates consistently through all releases instead of relying on "silent periods" as a judge of stability.
 
+## Release Process
```

Comment:
With 082ebc1 → 60c3c76 you've restored `project.md`, but you still need to drop the addition here.